src/unstract/adapters/embedding/palm/pyproject.toml

```diff
@@ -11,7 +11,6 @@ authors = [
     {name = "Zipstack Inc.", email = "devsupport@zipstack.com"},
 ]
 dependencies = [
```

Comment:
QQ: @gaya3-zipstack If we're not mentioning any dependency in such `pyproject.toml` files, is it still needed? I remember it was initially considered to try and keep them pluggable but I don't think we're following through properly
CC: @hari-kuriakose

Reply:
Yeah we could remove these. With llama-index 0.10.x, I don't think it is a good idea to maintain this anymore

Reply:
@hari-kuriakose your thoughts on this?

Reply:
Resolving conversations to allow merge - discussions can be continued on slack

Reply:
@chandrasekharan-zipstack  Unneeded dependencies can be removed. Any other use cases, we can discuss.